_source/_posts/2020-08-21-unity-csharp-games-security.md

```diff
@@ -36,20 +35,6 @@ The level of effort to build identity security from scratch is massive. Just to
 - Example code: [Native](https://github.com/nickgamb/Okta_Unity/blob/master/PanelManager.cs) / [OAuth](https://github.com/nickgamb/Okta_Unity/blob/master/OktaOAuth2.cs).
 - The Okta SDK Unity Package can be downloaded [here.](https://github.com/nickgamb/Okta_Unity/blob/master/OktaSDK.unitypackage).
 
-## Videos
-
-Follow along with our companion videos.
-
-NATIVE: [https://youtu.be/9C5xaaLt7Q4](https://youtu.be/9C5xaaLt7Q4)
-<div style="text-align: center">
-<iframe width="600" height="338" src="https://www.youtube.com/embed/9C5xaaLt7Q4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
-
-OAUTH: [https://youtu.be/ciDl0-Wtfvo](https://youtu.be/ciDl0-Wtfvo)
-<div style="text-align: center">
-<iframe width="600" height="338" src="https://www.youtube.com/embed/ciDl0-Wtfvo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
-
```

Comment:
Why remove?

Reply:
removed because they show the old UI and no CLI

Reply:
Added back in after chatting with @mraible in slack.